services/src/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/FlowHelperV2.groovy

```diff
@@ -172,6 +160,30 @@ class FlowHelperV2 {
         } || existingFlows*.flowId.contains(newFlow.flowId)
     }
 
+    /**
+     * Checks flow rules presence (or absence) on all involved switches.
+     */
+    void checkRulesOnSwitches(String flowId, int timeout, boolean rulesPresent) {
+        def flowEntry = db.getFlow(flowId)
+        def cookies = [flowEntry.forwardPath.cookie.value, flowEntry.reversePath.cookie.value]
+        def switches = PathHelper.convert(northbound.getFlowPath(flowEntry.flowId))*.switchId.toSet()
+        switches.each { sw ->
```

Comment:
This is fine, but as an improvement it would be good to call this in parallel, verify all the switches simultaneously

Reply:
will do in next pr